tempodb/encoding/vparquet3/schema.go

```diff
@@ -150,16 +150,27 @@ type DedicatedAttributes struct {
 	String10 *string `parquet:",snappy,optional,dict"`
 }
 
-type EventAttribute struct {
-	Key   string `parquet:",snappy,dict"`
-	Value []byte `parquet:",snappy"` // Was json-encoded data, is now proto encoded data
+type Event struct {
+	TimeSinceStartUnixNano uint64      `parquet:",delta"`
```

Comment:
The suffix `UnixNano` indicates nanoseconds since unix epoch. However, this field contains nanoseconds since the span start, therefore `TimeSinceStartNano` could a better name (not 100% sure about this)